CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Avoid prepending the same prefix twice to errors messages. (#423)
   This mostly happens with the `redis-rb 5+` gem as it translate `redis-client` errors.
+* Environment variable `SEMIAN_DISABLED` now skips creating a Semian resource. (#427)
+  Introduce environment variable `SEMIAN_CIRCUIT_BREAKER_DISABLED` to disable only
+  `circuit_breaker` and `SEMIAN_BULKHEAD_DISABLED` only `bulkhead`.
```

Comment:
So it's a bit nitpick territory, but common Ruby gems use an imperative stance for these variables:

  - `DISABLE_BOOTSNAP=1`
  - `DISABLE_BOOTSNAP_LOAD_PATH_CACHE=1`
  - `DISABLE_SPRING=1`
  - etc.

It might make sense to be consistent here.

Reply:
This would make it a breaking change, since those environment variable names already existed

Reply:
I missed these already existed. It's not worth renaming.